### ISSUE TITLE
A11y: chosen-single link should have role="button"

### ISSUE THREAD

**Issue by juanolalla** [JJJ/chosen#52]
Accessibility alerts about the following error: **Anchor element found with link content, but no href, ID or name attribute has been supplied.**

### Steps to reproduce

Just configure chosen and inspect the code around the drop-down item. Everything is wrapped in the "chosen-single" class link with no href, id or name.

### Expected behavior

The link `<a class="chosen-single">` should be a button if it is an action. Converting it into a button would have a great impact on the library, so I suggest just to add `role="button"` as attribute to this link.

**Comment by juanolalla** [comment]
Fixed in pull request: https://github.com/JJJ/chosen/pull/53

**Comment by juanolalla** [comment]
Added a commit to complete the implementation. Now it also adds an aria-label to inform screen readers what will happen if you click the button, "Show options" or "Hide options". It hides the visual arrow up/down indicator container from screen readers with aria-hidden.
